### ISSUE TITLE
key-spacing starts new "group" after empty line

### ISSUE THREAD

**Issue by thegranddesign** [eslint/eslint#10071]
**Tell us about your environment**

* **ESLint Version:** v4.17.0
* **Node Version:** v9.5.0
* **npm Version:** v5.6.0

**What parser (default, Babel-ESLint, etc.) are you using?**
babel-eslint

**Please show your full configuration:**
```js
    "key-spacing"                      : [
      "error",
      {
        "beforeColon" : false,
        "afterColon"  : true,
        "mode"        : "strict",
        "align"       : "value"
      }
    ],
```

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
import DS from 'ember-data';

export default DS.Model.extend({
  thingOne:   'one',
  thingTwo:   'two',

  thingThree: 'three',
});
```

```bash
eslint .
```

**What did you expect to happen?**
The values of all of the items in the object are aligned, therefore there should be no error.

**What actually happened? Please include the actual, raw output from ESLint.**
ESLint "resets" the "correct" position of the values if there is an empty line between "groups".

**Comment by thegranddesign** [comment]
I have tried just about every combination of options for the check and none of them fix it.

**Comment by thegranddesign** [comment]
Nevermind, I literally just figured out that `mode` needs to be set to `minimum`.
